### PR TITLE
Remove useless KnowhereConfig APIs

### PR DESCRIPTION
**Pull request by cydrain**

### CODE REVIEW

File: include/knowhere/comp/knowhere_config.h [hunk 1]
```diff
@@ -70,19 +70,6 @@ class KnowhereConfig {
...
-     */
-    static void
-    SetStatisticsLevel(const int32_t stat_level);
-
-    // todo: add log level?
-    /**
-     * set Log handler
-     */
-    static void
-    SetLogHandler();
```
Comment (PwzXxm) [inline include/knowhere/comp/knowhere_config.h:84]: Are these functions used in Milvus?

Comment (cydrain) [reply include/knowhere/comp/knowhere_config.h:84]: I'll remove them later

**Pull request merged**